test/suites/network_acl.sh

```diff
@@ -7,16 +7,23 @@ test_network_acl() {
   lxc network acl create testacl
   lxc project create testproj -c features.networks=true
   lxc project create testproj2 -c features.networks=false
+  lxc project create testproj3 -c features.networks=true
   lxc network acl create testacl --project testproj
+  lxc network acl create testacl2 --project testproj3
   lxc project show testproj | grep testacl # Check project sees testacl using it.
   ! lxc network acl create testacl --project testproj2 || false
   lxc network acl ls | grep testacl
   lxc network acl ls --project testproj | grep testacl
+  lxc network acl ls --project testproj3 | grep testacl2
   lxc network acl delete testacl
   lxc network acl delete testacl --project testproj
+  lxc network acl delete testacl2 --project testproj3
   ! lxc network acl ls | grep testacl || false
+  ! lxc network acl ls | grep testacl2 || false
   ! lxc network acl ls --project testproj | grep testacl || false
+  ! lxc network acl ls --project testproj3 | grep testacl2 || false
   lxc project delete testproj
+  lxc project delete testproj3
```

Comment:
I think for the system tests we're converging on a pattern of:
- When matching positively (checking something is there), use direct equality if possible (`[ "$(lxc ..)" = "<value>" ]`). If matching a substring then use `grep -F` where possible.
- When matching negatively, don't use grep as this can lead to erroneous tests where we're grepping for something that was never there (e.g. due to a typo), and instead use a line count or direct equality